test/lib-test/jdk/test/whitebox/vm_flags/SizeTTest.java

```diff
@@ -35,7 +35,7 @@
 import jdk.test.lib.Platform;
 
 public class SizeTTest {
-    private static final String FLAG_NAME = "ArrayAllocatorMallocLimit";
+    private static final String FLAG_NAME = "StringDeduplicationCleanupDeadMinimum";
```

Comment:
Again a non-experimental flag, like MetaspaceSize, might be better here.

Reply:
When setting `MetaspaceSize` to 0, the following assertion raised (at metaspcae.cpp: 316):
```C++
size_t MetaspaceGC::capacity_until_GC() {
  size_t value = Atomic::load_acquire(&_capacity_until_GC);
  assert(value >= MetaspaceSize, "Not initialized properly?");
  return value;
}
```
`LargePageSizeInBytes` is used instead.

Reply:
`MetaspaceSize` is used with its lower bound 65536. Otherwise, the test fails.